### PR TITLE
add precourse folder

### PR DESCRIPTION
**Pull request by edfrias**

### CODE REVIEW

File: staff/edgar-frias-jimenez/precourse/calculator-2/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+// Operands
```
Comment (carmecass) [inline staff/edgar-frias-jimenez/precourse/calculator-2/index.js:1]: Felicidades, muy buen código !

**Review by carmecass — COMMENTED** [review]
Felicitats, mot bé

File: staff/edgar-frias-jimenez/precourse/calculator/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    if (typeof newArgs[i] !== 'number' || newArgs[i] === null || newArgs[i] === undefined) {
+      isInvalid = true;
+    }
+  }
+
+  if (newArgs.length === 1 && isInvalid) {
+    console.log('Sorry you entered an invalid input');
+  } else if (newArgs.length === 1 && !isInvalid) {
+    const oneParamResult = Math.sqrt(newArgs[0]);
+    console.log(`As you only entered one input I can only give the square root of ${newArgs[0]}, that is: ${oneParamResult % 1 !== 0 ? oneParamResult.toFixed(3) : oneParamResult}`);
```
Comment (maz-e) [inline staff/edgar-frias-jimenez/precourse/calculator/index.js:16]: @edfrias repeat code in line 16, 23, 24, 25 and 26 - better in a function?

Ej: 
```javascript
function functionName(num) {
    return num % 1 !== 0 ? num.toFixed(3) : num;
}

const summatory = functionName(newArgs.reduce((a, b) => a + b));
```

Comment (edfrias) [reply staff/edgar-frias-jimenez/precourse/calculator/index.js:16]: @maz-e You are right! If you see in my next commit you'll see that I've created a new function that tries to see if the number has float or not and return the desired value, thanks mate!

**Pull request merged**